sandbox-start.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+# Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+set -euxo pipefail
+
+cd "$(dirname "${0}")"
+
+DAML_SRC=test/integration/src
+
+if [[ ! $(which daml) ]]; then
```

Comment:
Since we modify the PATH to add `~/.daml/bin` it would also be safe to check here whether `~/.daml/bin/daml` exists, otherwise the SDK keeps on being downloaded for no real reason.